index.html

```diff
@@ -187,14 +187,19 @@ <h2>Contact me</h2>
                 If you have an application you are interested in developing, 
                 a feature that you need built or a project that needs coding. 
                 i'd love to help with it.</p>
-            <form action="https://formspree.io/f/{form_id}" methop="post" class="text-me">
+            <form class="text-me" id="#form" method="post" action="https://formspree.io/f/xgedqoye">
                 <input type="text" class="simple" placeholder="User name" maxlength="30" required>
-                <input type="email" class="simple" placeholder="Email" required>
+                <input type="email" class="simple" placeholder="Email" id="e-mail" required>
+                <div class="formPopup">
+                    <p>e-mail must be all in lowercase!</p>
+                </div>
```

Comment:
As of the project's requirement the error have to be shown below the `textarea` input field and above the `Get in touch` button. Kindly, fix that.

Reply:
Thank you @Mre55 for reviewing my code and giving motivating feedback.

please can you make this required change  ***Make sure the person with less commit fix the issues for the sake of even contribution*** more clear to find the solution?

Reply:
@Mre55, Thank you!